src/solver/simulation/sim_structure_probleme_economique.h

```diff
@@ -39,7 +39,7 @@
 
 class AdequacyPatchRuntimeData;
 
-typedef struct
+struct CORRESPONDANCES_DES_VARIABLES
```

Comment:
Most of changes to remove warnings on Windows at compile time

Reply:
Yes I noticed them as well